secboot/secboot_tpm.go

```diff
@@ -63,17 +63,13 @@ var (
 
 	randutilRandomKernelUUID = randutil.RandomKernelUUID
 
-	isTPMEnabled = isTPMEnabledImpl
-	provisionTPM = provisionTPMImpl
+	isTPMEnabled           func(tpm *sb_tpm2.Connection) bool                                                  = (*sb_tpm2.Connection).IsEnabled
+	sbTPMEnsureProvisioned func(tpm *sb_tpm2.Connection, mode sb_tpm2.ProvisionMode, lockoutAuth []byte) error = (*sb_tpm2.Connection).EnsureProvisioned
```

Comment:
Question: why is the type needed? Wouldn't `isTPMEnabled = (*sb_tpm2.Connection).IsEnabled` work as well?

Reply:
I do recall it used to be needed, but may it's not anymore.